api.md

```diff
@@ -170,6 +172,75 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## add_data_node() [](add_data_node)
+
+Add a new data node to the database to be used for creating
+distributed hypertables.
+
+Newly created distributed hypertables will be able to make use of this
+data node for distributing data across the cluster. Note that existing
```

Comment:
```suggestion
data node for distributing data across the distributed database. Note that existing
```
As mentioned in another PR it is better to avoid `cluster` due to confusion with PostgreSQL terminology.

Reply:
Changed this to "apportioning data across the distributed database" to make it less repetitive.